mujoco_py/tests/test_cymj.py

```diff
@@ -665,8 +665,8 @@ def func(sim, event):
         img1 = sim.render(width=40, height=40, camera_name="camera1")
         img2 = sim.render(width=40, height=40, camera_name="camera2")
         try:
-            assert np.sum(img1[:]) == 23255
-            assert np.sum(img2[:]) == 12007
+            assert np.sum(img1[:]) in range(23251, 23255+1)
+            assert np.sum(img2[:]) in range(12001, 12007+1)
```

Comment:
I think this should not change -- we want the test to be exact.

Also this test is expected to be exact only running on our automated test platform, on other platforms (given variations in GL libraries) the match is inexact.

Does this test fail (without this change) when you run `make test`?  (Which runs tests in a dockerfile)

Reply:
No one has run this test since 22415daf turned it off on all platforms, so I'm not sure why it matters what the number is on Linux where it isn't run.

Reply:
I'm happy to revert the number, though that will just mean the test (1) fails on my machine and (2) continues to be skipped by `make test` (assuming I understand correctly that Docker is using Linux).

Reply:
@girving good catch!!

 does the test pass on your machine with the new values? in that case it's probably fine to set them to exact equal `23251` and `12001`. indeed, something might have changed since the test worked last. i also don't like this test super much, since i don't think there are any guarantees that it will hold across MacOS upgrades etc.